--- ldc-source/runtime/druntime/src/core/thread/osthread.d	2023-06-07 23:30:53.768628603 +0200
+++ ldc-source/runtime/druntime/src/core/thread/osthread.d	2023-06-07 23:31:14.928644408 +0200
@@ -1437,6 +1437,7 @@
 package extern(D) void callWithStackShell(scope callWithStackShellDg fn) nothrow
 in (fn)
 {
+    assert(0);
     // The purpose of the 'shell' is to ensure all the registers get
     // put on the stack so they'll be scanned. We only need to push
     // the callee-save registers.
